seed/csharp-sdk/grpc-proto-exhaustive/read-only-memory/src/SeedApi/Dataservice/DataserviceClient.cs

```diff
@@ -22,6 +22,23 @@ internal DataserviceClient(RawClient client)
         _dataService = new DataService.DataServiceClient(_grpc.Channel);
     }
 
+    /// <example>
+    /// <code>
+    /// await client.Dataservice.UploadAsync(
+    ///     new UploadRequest
+    ///     {
+    ///         Columns = new List<Column>()
+    ///         {
+    ///             new Column
+    ///             {
+    ///                 Id = "id",
+    ///                 Values = new List<float>() { 1.1 },
```

Comment:
I would have expected this to be a `ReadOnlyMemory<float>`. https://github.com/fern-api/fern/blob/6bd9d9a2c17d4e30c9e96ad354b691fbbd8fba5f/seed/csharp-sdk/grpc-proto-exhaustive/read-only-memory/src/SeedApi/Types/Column.cs#L14

Is there any chance that the snippet generator is using a different context? This should be handled by the writer automatically.

Reply:
fixed, great catch!